nameko_eventlog_dispatcher/eventlog_dispatcher.py

```diff
@@ -0,0 +1,124 @@
+import logging
+from copy import deepcopy
+from datetime import datetime, timezone
+
+from kombu.common import maybe_declare
+from nameko.amqp import get_connection, verify_amqp_uri
+from nameko.events import EventDispatcher
+from nameko.rpc import Rpc
+from nameko.standalone.events import event_dispatcher, get_event_exchange
+from nameko.web.handlers import HttpRequestHandler
+
+
+log = logging.getLogger(__name__)
+
+
+class EventLogDispatcher(EventDispatcher):
+
+    """Dispatcher of event logs reusing the Nameko `EventDispatcher`.
+
+    When the auto capture feature is enabled, it dispatches an event
+    each time an entrypoint is fired.
+
+    Also provides the ability to manually dispatch event logs,
+    optionally providing related event data.
+
+    On both cases, relevant worker context information is added to the
+    event data.
+    """
+
+    ENTRYPOINT_FIRED = 'entrypoint_fired'
```

Comment:
I don't think it should be necessary to subclass just to change some values which could easily be made configurable. I'm happy to leave this discussion for a separate PR though.